unstructured/ingest/evaluate.py

```diff
@@ -152,8 +152,8 @@ def measure_element_type_accuracy_command(
     help="Directory to save the output evaluation metrics to. Default to \
         your/working/dir/metrics/",
 )
-def group_text_extraction_accuracy_command(grouping: str, data_input: str, export_dir: str):
-    return group_text_extraction_accuracy(grouping, data_input, export_dir)
+def get_mean_grouping_command(grouping: str, data_input: str, export_dir: str):
+    return get_mean_grouping(grouping, data_input, export_dir)
```

Comment:
Shouldn't we expose `metric_strategy` as well? From what I see in `get_mean_grouping` definition, `metric_strategy` is of type `str`, not `Optional[str]`. So this call is invalid (?).

Reply:
Missed that! Thank you.